core/src/main/java/org/acegisecurity/AccessDeniedException.java

```diff
@@ -22,21 +22,22 @@
  * THE SOFTWARE.
  */
 
-package hudson.security;
+package org.acegisecurity;
 
-import jenkins.security.NonSerializableSecurityContext;
-import org.acegisecurity.Authentication;
+import hudson.security.AccessDeniedException2;
 
 /**
- * @deprecated use {@link NonSerializableSecurityContext} instead
+ * @deprecated use {@link AccessDeniedException2}
  */
 @Deprecated
-public class NotSerilizableSecurityContext extends NonSerializableSecurityContext {
+public class AccessDeniedException extends org.springframework.security.access.AccessDeniedException {
```

Comment:
Git trying to be too clever. Deleting long-deprecated typo `NotSerilizableSecurityContext` (not used by plugins); and adding a façade for `AccessDeniedException`.